src/components/Dashboard/Tiles/Apps/AppTile.tsx

```diff
@@ -20,49 +20,71 @@ export const AppTile = ({ className, app }: AppTileProps) => {
 
   const { cx, classes } = useStyles();
 
+  const { colorScheme } = useMantineTheme();
+
   const {
     classes: { card: cardClass },
   } = useCardStyles(false);
 
   function Inner() {
     return (
-      <>
-        <Stack
+      <Tooltip.Floating
+        label={app.behaviour.tooltipDescription}
+        position="right-start"
+        c={ colorScheme === 'light' ? "black" : "dark.0" }
+        color={ colorScheme === 'light' ? "gray.2" : "dark.4" }
+        multiline
+        disabled={!app.behaviour.tooltipDescription}
+        styles={{ tooltip: { '&': { maxWidth: 300, }, }, }}
```

Comment:
Do you really need ``'&'`` here?

Reply:
Yes. max-width didn't work on the tooltip and fixed width would force a bigger box than necessary on shorter text.
Here it targets the child text element of the tooltip on which the max-width option works and let the tooltip box wrap around it.